content/it/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -12,7 +12,7 @@ Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concep
 {{% capture body %}}
 ## Planning a cluster
```

Comment:
"Planning a cluster" -> "Progettare un cluster" (1) or "Pianificare un cluster" (2)

I think option (1) is better, option (2) is still acceptable.